apps/02RockPaperScissors.js

```diff
@@ -5,16 +5,42 @@ var prompt = require('prompt');
 prompt.start();
 
 
+
 function rockPaperScissors(hand1, hand2) {
 
-    // Write code here
+    hand1 = hand1.toLowerCase();
+    hand2 = hand2.toLowerCase();
+    hand1 = hand1.trim();
+    hand2 = hand2.trim();
+
+    var handWin1 = 'Hand one wins!';
+    var handWin2 = 'Hand two wins!';
+    var handTie = "It's a tie!";
 
```

Comment:
Good use of variables here! It will save you the trouble of having to copy/paste 'Hand one wins!" a bunch of times.